.github/workflows/update-engines-version.yml

```diff
@@ -40,6 +40,8 @@ jobs:
         run: |
           echo 'Checking that @prisma/engines-version have the published version @${{ github.event.inputs.version }}'
           pnpm --package=@prisma/ensure-npm-release dlx enr update -p @prisma/engines-version -u ${{ github.event.inputs.version }}
+          echo 'Checking that @prisma/prisma-fmt-wasm have the published version @${{ github.event.inputs.version }}'
+          pnpm --package=@prisma/ensure-npm-release dlx enr update -p @prisma/prisma-fmt-wasm -u ${{ github.event.inputs.version }}
```

Comment:
I'm keeping two different echos, as @janpio suggested

Reply:
`pnpm --package=@prisma/ensure-npm-release dlx enr update -p @prisma/prisma-fmt-wasm -u ${{ github.event.inputs.version }}` will wait for the latest published version of `@prisma/prisma-fmt-wasm` on the `npm` public registry to be `${{ github.event.inputs.version }}` before continuing. This solves the "race condition" issue we have arised internally.